src/sentry/api/endpoints/organization_events_distribution.py

```diff
@@ -56,14 +56,14 @@ def get(self, request, organization):
             referrer="api.organization-events-distribution",
         )["data"]
 
-        projects = {p.id: p.slug for p in self.get_projects(request, organization)}
```

Comment:
@fpacifici Projects was being fetched on every tag; wondering if this was related to some concerns you had last week. I've since moved this.